crates/ide-assists/src/handlers/generate_delegate_methods.rs

```diff
@@ -183,6 +187,12 @@ pub(crate) fn generate_delegate_methods(acc: &mut Assists, ctx: &AssistContext<'
                 let assoc_items = impl_def.get_or_create_assoc_item_list();
                 assoc_items.add_item(f.clone().into());
 
+                PathTransform::generic_transformation(
+                    &ctx.sema.scope(strukt.syntax()).unwrap(),
+                    &ctx.sema.scope(method_source.syntax()).unwrap(),
+                )
+                .apply(f.syntax());
```

Comment:
```suggestion
                if let Some((target, source)) =
                    ctx.sema.scope(strukt.syntax()).zip(ctx.sema.scope(method_source.syntax()))
                {
                    PathTransform::generic_transformation(&target, &source).apply(f.syntax());
                }
```
The unwraps shouldn't really be possible to hit here, but we are trying to not use unwraps in assist code if possible.

Reply:
Ah okay, sorry about that!